sysutils/edid-decode/Portfile

```diff
@@ -0,0 +1,17 @@
+PortSystem       1.0
+
+name             edid-decode
+version          git-e48fb38
+categories       sysutils
+platforms        darwin
+maintainers      @wwalexander
+license          MIT
+description      Decode EDID data in human-readable format
+long_description edid-decode decodes EDID monitor description data in human-readable format
+homepage         https://git.linuxtv.org/edid-decode.git
```

Comment:
You can probably do something like:

```
set domain  linuxtv.org
homepage    https://git.${domain}/edid-decode.git
git.url     git://${domain}/edid-decode.git
```

Reply:
Should I also use ${name} in these URLs, e.g.

```
set domain  linuxtv.org
homepage    https://git.${domain}/${name}.git
git.url     git://${domain}/${name}.git
```

or would that be a bad practice?

Reply:
I added this (d73a890), using `${name}`; let me know if I should revise.